specs/WebResourceResponseReceived.md

```diff
@@ -0,0 +1,219 @@
+# Background
+The WebView2 team has been asked for an API to get the response for a web
+resource as it was received and to provide request headers not available when
+`WebResourceRequested` event fires (such as Authentication headers). The
+`WebResourceResponseReceived` event provides such HTTP response representation
+and exposes the request as sent over the wire.
+
+In this document we describe the new API. We'd appreciate your feedback.
+
+# Description
+The `WebResourceResponseReceived` event fires when the WebView receives the
+response for a request for a web resource. It provides access to both the
+response as it was received and the request as it was sent over the wire,
+including modifications made by the network stack (such as adding of
```

Comment:
such as **the** adding of Authorization headers

Reply:
👍